docs/csharp/getting-started/index.md

```diff
@@ -1,51 +1,47 @@
 ---
-title: Get Started - C# Guide
-description: Find short, simple tutorials that let you quickly learn introductory C# concepts and write .NET Core applications.
-helpviewer_keywords:
-  - "Visual C#, getting started"
-  - "getting started, Visual C#"
-ms.date: 04/05/2019
+title: Get Started - an introduction to the C# language and .NET"
+description: Learn the basics of C# and .NET. Get an overview of the C# language and .NET ecosystem.
+ms.date: 10/13/2020
```

Comment:
Are the helpviewer_keywords intentionally removed?
I assume they have something to do with SEO

Reply:
They were intentionally removed. They were historically used for search in the offline CHM versions of Visual Studio help. They aren't needed any more.